src/AWS.Deploy.CLI/Commands/TypeHints/BeanstalkEnvironmentCommand.cs

```diff
@@ -20,13 +24,24 @@ public BeanstalkEnvironmentCommand(IAWSResourceQueryer awsResourceQueryer, ICons
             _consoleUtilities = consoleUtilities;
         }
 
-        public async Task<object> Execute(Recommendation recommendation, OptionSettingItem optionSetting)
+        private async Task<T> GetData<T>(Recommendation recommendation, OptionSettingItem optionSetting)
         {
-            var currentValue = recommendation.GetOptionSettingValue(optionSetting);
             var applicationOptionSetting = recommendation.GetOptionSetting(optionSetting.ParentSettingId);
-
             var applicationName = recommendation.GetOptionSettingValue(applicationOptionSetting) as string;
             var environments = await _awsResourceQueryer.ListOfElasticBeanstalkEnvironments(applicationName);
+            return JsonConvert.DeserializeObject<T>(JsonConvert.SerializeObject(environments));
+        }
+
+        public async Task<List<TypeHintResource>?> GetServerModeResource(Recommendation recommendation, OptionSettingItem optionSetting)
```

Comment:
I don't like calling these `ServerMode` implying they couldn't be used for anything else. Can we just call this method `GetResources`?